cime/src/drivers/mct/main/prep_glc_mod.F90

```diff
@@ -607,6 +777,156 @@ end subroutine prep_glc_map_one_state_field_lnd2glc
 
   !================================================================================================
 
+  subroutine prep_glc_calculate_subshelf_boundary_fluxes
+
+    !---------------------------------------------------------------
+    ! Description
+    ! On the ice sheet grid, calculate shelf boundary fluxes
+
+    use shr_const_mod , only: SHR_CONST_KAPPA_LAND_ICE
+
+    ! Local Variables
+
+    integer :: gsize, n, egi
+    type(mct_aVect), pointer :: o2x_ox ! Ocn export, ocn grid, cpl pes
+    type(mct_aVect), pointer :: x2g_gx ! Glc import, glc grid, cpl pes
+    type(mct_aVect), pointer :: g2x_gx ! Glc import, glc grid, cpl pes
+
+    integer :: index_x2g_So_blt
+    integer :: index_x2g_So_bls
+    integer :: index_x2g_So_htv
+    integer :: index_x2g_So_stv
+    integer :: index_x2g_So_rhoeff
+    integer :: index_g2x_Sg_tbot
+    integer :: index_g2x_Sg_dztbot
+    integer :: index_g2x_Sg_lithop
+    integer :: index_g2x_Sg_icemask_floating
+
+    integer :: index_g2x_Sg_blis
+    integer :: index_g2x_Sg_blit
+    integer :: index_g2x_Fogx_qiceho
+    integer :: index_g2x_Fogx_qicelo
+    integer :: index_x2g_Fogx_qiceli
+    integer :: index_x2g_Fogx_qicehi
+
+    character(*), parameter :: subname = '(prep_glc_calculate_subshelf_boundary_fluxes)'
+    !---------------------------------------------------------------
+
+    if (.not.(glc_present)) return
+    !write(logunit,*) 'Starting prep_glc_calculate_subshelf_boundary_fluxes'
```

Comment:
Please remove commented-out lines of code like this.

Reply:
Fixed in 733a46925b5580e5888bdfa417add9916e7cf2df